packages/lib-classifier/src/plugins/drawingTools/components/Point/Point.js

```diff
@@ -22,7 +22,7 @@ function Point ({ active, children, mark, scale }) {
   const radius = RADIUS[size] / scale
 
   return (
-    <g transform={`translate(${mark.x}, ${mark.y})`}>
```

Comment:
Why is this removed?

Reply:
It's been a while since I wrote this but it should have moved to the drawing tool root.